src/library/Button/constants.js

```diff
@@ -16,6 +16,9 @@ export const SIZE = {
 };
 
 export const VARIANT = {
+  /* TargetX Custom Variant */
+  greyscale: 'greyscale',
```

Comment:
Let's stick with the American spelling `grayscale`. I think that's what's already being used my mineral-ui.

Also, where do we define the color ramp for it?